yarn-project/noir-contracts/src/contracts/token_bridge_contract/src/util.nr

```diff
@@ -107,20 +74,11 @@ fn get_withdraw_content_hash(recipient: Field, amount: Field, callerOnL1: Field)
         hash_bytes[i + 36] = recipient_bytes[i];
         hash_bytes[i + 68] = callerOnL1_bytes[i];
     }
-    let content_sha256 = sha256(hash_bytes);
-
-    // Convert the content_sha256 to a field element
-    let mut v = 1;
-    let mut high = 0 as Field;
-    let mut low = 0 as Field;
-
-    for i in 0..16 {
-        high = high + (content_sha256[15 - i] as Field) * v;
-        low = low + (content_sha256[16 + 15 - i] as Field) * v;
-        v = v * 256;
-    }
+    let content_hash = bytes_to_sha256_in_a_field(hash_bytes);
+    content_hash
+}
 
-    // Abuse that a % p + b % p = (a + b) % p and that low < p
-    let content = low + high * v;
-    content
+fn compute_secret_hash(secret: Field) -> Field {
```

Comment:
Might fit better in with the hash.nr lib?

Reply:
good call